ci/install.sh

```diff
@@ -7,9 +7,9 @@
 set -o errexit \
     -o pipefail
 
-wget https://repo.continuum.io/miniconda/Miniconda3-4.7.12.1-Linux-x86_64.sh \
-    --output-document miniconda.sh
-bash miniconda.sh -b -p $HOME/miniconda
+wget https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge-pypy3-$(uname)-$(uname -m).sh \
```

Comment:
Note that this line won't work for Windows.  Locally I get
```
$ uname
MINGW64_NT-10.0-19043
```
instead of `Windows`.

I don't think that matters though because this script is only run in AppVeyor on Linux.